Dockerfile

```diff
@@ -25,19 +32,28 @@ RUN apt-get update -y \
     make \
     netcat-openbsd \
     perl \
-    postgresql-13 \
-    procps \
     sudo \
     tar \
     wget \
     xz-utils \
     zip \
     zlib1g-dev \
-  && apt-get autoremove \
-  && mkdir -p /tmp/stack \
-  && cd /tmp/stack \
-  && wget --output-document stack.tgz --no-verbose "https://github.com/commercialhaskell/stack/releases/download/v$STACK_VERSION/stack-$STACK_VERSION-linux-x86_64.tar.gz" \
-  && tar --extract --file stack.tgz --strip-components 1 --wildcards '*/stack' \
-  && mv stack /usr/local/bin/ \
-  && cd - \
-  && rm -r /tmp/stack
+  && apt-get autoremove
+
+# next build docker-stack develop after below should create /stack with right perms
+
+RUN mkdir -p /stack && chown -R $USER:$USER stack
+
+USER "$USER"
+
+# TODO building from scratch work without this because chown 1000 takes care of it in COPY?
+# RUN mkdir -p /home/haskell/.stack/global-project/
+
+# Copy stack.yaml with overriden packages and extra-deps
+COPY --chown=1000:1000 config.yaml /home/haskell/.stack/config.yaml
+COPY --chown=1000:1000 stack.yaml /home/haskell/.stack/global-project/stack.yaml
```

Comment:
Should these `chown`s use `$UID` somewhere? We don't have an `ARG` for `GID`, but we might need one.